lib/oelite/compat.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+import os
+import fcntl
+import ctypes
+import datetime
+import sys
+
+__all__ = ["dup_cloexec", "open_cloexec"]
+
+uname = os.uname()
+sysname, release, machine = (uname[0], uname[2], uname[4])
```

Comment:
Well, at least it would have to be `sysname, _, release, _, machine = os.uname() ` to throw away `uname[1]` and `uname[3]`. That's a standard python idiom, but I don't write enough snake to think of writing stuff that way, and for the same reason I also have to think twice when I read code like that.